app/uk/gov/hmrc/exports/services/mapping/goodsshipment/GoodsShipmentBuilder.scala

```diff
@@ -40,7 +41,9 @@ class GoodsShipmentBuilder @Inject()(
   override def buildThenAdd(exportsCacheModel: ExportsDeclaration, declaration: Declaration): Unit = {
     val goodsShipment = new GoodsShipment()
 
-    exportsCacheModel.natureOfTransaction.foreach(goodsShipmentNatureOfTransactionBuilder.buildThenAdd(_, goodsShipment))
+    if (exportsCacheModel.`type` != DeclarationType.SIMPLIFIED) {
```

Comment:
I think it'll be more readable if we will check what journey can have those informations. I know the if will be longer but it will explain more if we will check Standard and Supplemetary instead of checking different than Simplified